.github/workflows/ci_publish.yml

```diff
@@ -21,7 +21,7 @@ env:
 jobs:
 
   publish_html:
-    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' && github.repository == 'Caltech-IPAC/irsa-tutorials' }}
```

Comment:
Let me investigate this one a bit more as normally we turn off fork-based runs within the `on:` section above rather than in individual conditionals (we don't even want this to trigger a skipped job). 

(I didn't picked up on this bug as somehow jobs are not triggered on my own fork, but I see both you and Jaladh had them).